<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-LPSVTKP" failures="1" tests="2" name="Default test(failed)(failed)" time="0.247" errors="0" timestamp="2022-11-22T09:43:18 IST">
  <testcase classname="failed_Xml.Failed_xml_ex2" name="C" time="0.013"/>
  <testcase classname="failed_Xml.Failed_xml_ex2" name="B" time="0.125">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method failed_Xml.Failed_xml_ex2.B() didn&amp;apos;t finish within the time-out 100">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failed_Xml.Failed_xml_ex2.B() didn't finish within the time-out 100
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- B -->
  <testcase classname="failed_Xml.Failed_xml_ex2" name="f" time="0.0">
    <ignored/>
  </testcase> <!-- f -->
  <testcase classname="failed_Xml.Failed_xml_ex2" name="A" time="0.0">
    <ignored/>
  </testcase> <!-- A -->
</testsuite> <!-- Default test(failed)(failed) -->
